### PR TITLE
Portfolio Arianna

### PR DESCRIPTION
**Pull request by ariallahyar**
The content is just placeholder for the time being. The structure and styling is complete.

Link to my project live: https://ariallahyar.netlify.app/

### CODE REVIEW

File: code/index.html [hunk 1]
```diff
@@ -0,0 +1,229 @@
...
+			<div class="contact-text-group">
+				<h2>Contact</h2>
+				<p>Arianna Allahyar</p>
+				<a href="mailto:anallahyar@gmail.com">anallahyar@gmail.com</a>
+			</div>
+		</div>
+	</footer>
+</body>
+
+</html>
```
Comment (savannah-hayes) [inline code/index.html:229]: Great job using semantic HTML throughout your entire project.

File: code/index.html [hunk 2]
```diff
@@ -0,0 +1,229 @@
...
+				</div>
+			</div>
+		</section>
+	</main>
+	<footer>
+		<div class="container">
+			<div class="contact-text-group">
+				<h2>Contact</h2>
+				<p>Arianna Allahyar</p>
+				<a href="mailto:anallahyar@gmail.com">anallahyar@gmail.com</a>
```
Comment (savannah-hayes) [inline code/index.html:223]: What does this href_mailto do? I don't think I have ever seen this before.

File: code/index.html [hunk 3]
```diff
@@ -0,0 +1,229 @@
...
+	<link rel="stylesheet" href="./style.css" />
+	<link rel="preconnect" href="https://fonts.googleapis.com">
+	<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+	<link
+		href="https://fonts.googleapis.com/css2?family=Montserrat:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&display=swap"
+		rel="stylesheet" />
+	<link
+		href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap"
+		rel="stylesheet">
+
```
Comment (savannah-hayes) [inline code/index.html:17]: This is just a small thing, but this seems to be indent spaced with 4 spaces. I believe Technigo wants us to use 2 indent spaces.

File: code/index.html [hunk 4]
```diff
@@ -0,0 +1,229 @@
...
+	<header class="header">
+		<div class="container header-container">
+			<div class="header-socials-groups">
+				<a href="https://www.linkedin.com/in/ariannaallahyar/">
+					<img class="header-social-icon" src="./assets/linkedin_white.svg" alt="LinkedIn logo">
+				</a>
+				<a href="https://github.com/ariallahyar">
+					<img class="header-social-icon" src="./assets/github_white.svg" alt="GitHub logo">
+				</a>
+			</div>
```
Comment (savannah-hayes) [inline code/index.html:31]: Good job adding the content in the alt tags, from an accessibility stand point.

**Review by savannah-hayes — APPROVED** [review]
Your portfolio and your code looks great. I apologize for not having more comments, your code is clean and very easy to follow. There are no bugs in your code and everything works smoothly.

From what I can see it follows all the requirements. I only had small details to point out and overall you did a great job!

File: code/index.html (hunk 2 above)
Comment (ariallahyar) [reply code/index.html:223]: When the email address is clicked, a new email message will open in whatever default email sender is on a users computer with the "To: ___" already filled in with the email.

**Pull request closed without merging**